backend/python/app/services/implementations/story_service.py

```diff
@@ -222,26 +231,29 @@ def update_story_translation_contents(self, story_translation_contents):
 
     def get_story_translations_available_for_review(self, language, level):
         try:
-            return (
-                db.session.query(
-                    Story.id.label("story_id"),
-                    Story.title.label("title"),
-                    Story.description.label("description"),
-                    Story.youtube_link.label("youtube_link"),
-                    Story.level.label("level"),
-                    StoryTranslation.id.label("id"),
-                    StoryTranslation.language.label("language"),
-                    StoryTranslation.stage.label("stage"),
-                    StoryTranslation.translator_id.label("translator_id"),
-                    StoryTranslation.reviewer_id.label("reviewer_id"),
-                )
-                .join(StoryTranslation, Story.id == StoryTranslation.story_id)
+            stories = (
+                Story.query.join(StoryTranslation, Story.id == StoryTranslation.story_id)
                 .filter(Story.level <= level)
                 .filter(StoryTranslation.language == language)
                 .filter(StoryTranslation.reviewer_id == None)
+                .order_by(Story.id)
                 .all()
             )
 
+            story_translations = (
+                StoryTranslation.query.join(Story, Story.id == StoryTranslation.story_id)
+                .filter(Story.level <= level)
+                .filter(StoryTranslation.language == language)
+                .filter(StoryTranslation.reviewer_id == None)
+                .order_by(Story.id)
+                .all()
+            )
+
+            for i in range(len(stories)):
+                stories[i] = {**stories[i].to_dict(), **story_translations[i].to_dict(include_relationships=True)}
```

Comment:
drive by comment: something like this might fit your situation
https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression-taking-union-of-dictiona